### PR TITLE
feat(markdown): keep meta from fenced code block

### PR DESCRIPTION
**Pull request by Qwertovsky**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Don't drop a info from fenced code
````markdown
```javascript[file.js]{4-6,7} meta-info=val
export default () => {
  console.log('Code block')
}
```
````
So `meta-info=val` will be accessible from the document tree after parsing. Property `meta` will be next to `language`, `filename`, `highlights`.
It can help to do other modifications with remark plugins, transformers, or ProseCode component.

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by farnabaz — APPROVED** [review]
LGTM 👍

**Comment by farnabaz** [comment]
@Qwertovsky do you mind fixing linter issues?

File: test/features/parser-markdown.ts [hunk 1]
```diff
@@ -39,6 +39,31 @@ export const testMarkdownParser = () => {
...
+    test('Keep meta from fenced code block', async () => {
+      const parsed = await $fetch('/api/parse', {
+        method: 'POST',
+        body: {
+          id: 'content:index.md',
+          content: [
+          '``` ts [file.ts]{4-6,7} other code block info',
+          'let code = undefined;',
+          'return code;',
+          '```'
```
Comment (nobkd) [inline test/features/parser-markdown.ts:51]:
```suggestion
            '```ts [file.ts]{4-6,7} other code block info',
            'let code = undefined;',
            'return code;',
            '```'
```

**Review by nobkd — CHANGES_REQUESTED** [review]

**Comment by SimonScholz** [comment]
The sample mentioned above using `{4-6,7}` is not targeting the sample code since it has only 3 lines.

image

This is really unfortunate because I am currently facing the issue that no different highlighting is applied to the lines in brackets, e.g., `{4-6,7}` 
Shall I open a bug for this or is the `{4-6,7}` somehow not supported?

**Comment by nobkd** [comment]
@SimonScholz
Thos should be fixed with nuxt-modules/mdc#21 and will be fixed when the package is updated in Content. If that's the case, then the edge channel should already have the fix. Otherwise it should be fixed when a new version of Content gets released

**Pull request merged**